src/actions/model-actions.js

```diff
@@ -94,6 +94,24 @@ const remove = (model, index) => (dispatch, getState) => {
   });
 };
 
+const move = (model, indexFrom, indexTo) => (dispatch, getState) => {
+  const collection = _get(getState(), model, []);
+
+  if (indexFrom >= collection.length || indexTo >= collection.length) {
+    return;
```

Comment:
Yeh - you're right - I think it would be better to throw an error with a helpful message.  Will update later. Looks like there are now merge conflicts that need resolving anyway so I'll fix up later.